src/sdkRuntimeModels.ts

```diff
@@ -116,59 +120,76 @@ export interface SDKProduct {
 }
 
 export interface MParticleWebSDK {
-    addForwarder(mockForwarder: any);
+    addForwarder(mockForwarder: MPForwarder): void;
     Identity: SDKIdentityApi;
     Logger: SDKLoggerApi;
     _Store: SDKStoreApi;
     _Helpers: SDKHelpersApi;
-    config: SDKConfig;
+    config: SDKInitConfig;
     _resetForTests(MPConfig: SDKConfig): void;
-    init(apiKey: string, config: SDKConfig): void;
+    init(apiKey: string, config: SDKInitConfig, instanceName?: string): void;
     getInstance();
     ServerModel();
     upload();
     setPosition(lat: number | string, lng: number | string): void;
-    logEvent(eventName: string, eventType?: number, attrs?: { [key: string]: string }): void;
+    logEvent(
+        eventName: string,
+        eventType?: number,
+        attrs?: { [key: string]: string }
+    ): void;
```

Comment:
this should have customFlags as a 4th argument

```suggestion
        attrs?: { [key: string]: string },
        customFlags?: { [key: string]: string }
    ): void;
```